class.jetpack-debugger.php

```diff
@@ -18,6 +18,31 @@ private static function is_jetpack_support_open() {
 		}
 	}
 
+	private static function what_jetpack_plan() {
+		$plan = Jetpack::get_active_plan();
+		$plan = $plan['product_slug'];
+		switch ( $plan ){
+			case 'jetpack_free' :
+			default :
+				$plan = 'Free';
+				break;
+			case 'jetpack_premium' :
+			case 'jetpack_premium_monthly' :
+				$plan = 'Premium';
+				break;
+			case 'jetpack_personal' :
```

Comment:
Just the plan that the site thinks it has is enough for us. I had Personal first for performance (started this with if statements and Personal, I'd expect, to be the most purchased plan.

The KISS update negates the need for that though.